### PR TITLE
Init boot application

### PR DESCRIPTION
**Pull request by skawhdgjs**

### CODE REVIEW

File: jongheon/src/main/java/com/study/jong/persistence/ch002/entity/MemberEntity.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.study.jong.persistence.ch002.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+// Table annotation이 없으면 기본적으로 entity이름을 상요
```
Comment (kksshh525) [inline jongheon/src/main/java/com/study/jong/persistence/ch002/entity/MemberEntity.java:14]: I think it has a typo.

File: jongheon/src/main/resources/sql/0001_CreateMemberTable.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE MEMBER (
+    ID VARCHAR(255) NOT NULL,
+    NAME VARCHAR(255),
+    AGE INTEGER,
+    PRIMARY KEY (ID)
+)
```
Comment (kksshh525) [inline jongheon/src/main/resources/sql/0001_CreateMemberTable.sql:6]: It needs to add newline.

File: jongheon/src/main/resources/sql/0001_CreateMemberTable.sql [hunk 3]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE MEMBER (
+    ID VARCHAR(255) NOT NULL,
+    NAME VARCHAR(255),
+    AGE INTEGER,
```
Comment (kksshh525) [inline jongheon/src/main/resources/sql/0001_CreateMemberTable.sql:4]: Is there any reason using upper case?

Comment (skawhdgjs) [reply jongheon/src/main/resources/sql/0001_CreateMemberTable.sql:4]: This book's convention

File: jongheon/src/main/resources/sql/0001_CreateMemberTable.sql (hunk 2 above)
Comment (skawhdgjs) [reply jongheon/src/main/resources/sql/0001_CreateMemberTable.sql:6]: Don't bother me kk

File: jongheon/src/main/java/com/study/jong/persistence/ch002/entity/MemberEntity.java (hunk 1 above)
Comment (skawhdgjs) [reply jongheon/src/main/java/com/study/jong/persistence/ch002/entity/MemberEntity.java:14]: Correct when i present

**Pull request merged**